docs/orchestration/flow_config/storage.md

```diff
@@ -273,6 +276,15 @@ storage = Git(
 )
 ```
 
+:::tip Git Deploy Keys
+To use `Git` storage with Deploy Keys, ensure your environment is configured to use Deploy Keys. Then, create a `Git` storage class with `use_ssh=True`.
```

Comment:
Does this mean that if I use something like a K8s agent I have to setup my deploy key in my agent container? Why can't I use a Prefect Secret for that value?

Reply:
Yes you would have to configure it in the container (or use access tokens as a Secret).

Using a Secret for that value would involve manipulation of ssh configs, which I think is outside of Prefect's scope

Reply:
Ah so the confusing part here is gitlab has deploy _tokens_

https://docs.gitlab.com/ee/user/project/deploy_tokens/#git-clone-a-repository

Reply:
Haha yes I was confused about this as well. Deploy Tokens will work like other access tokens without issues

Reply:
If you don't add a note about that we're going to get some questions haha, just this one last thing and this is good to merge :D

Reply:
Added a note and example here https://github.com/PrefectHQ/prefect/pull/4418/commits/9ada00c0950984503385fb211a2a297750b587f2#diff-357fea1f44b9d331a93957d9d6b6408d2ffc07866872dfff6f91f2bb7af3a83cR288